rmgpy/data/thermo.py

```diff
@@ -1975,12 +1975,17 @@ def compute_group_additivity_thermo(self, molecule):
                     logging.error(molecule.to_adjacency_list())
                     raise
                 if not data_added:
-                    neighbors = ''.join(sorted([atom2.atomtype.label for atom2 in atom.edges.keys()]))
-                    if thermo_data.comment:
-                        thermo_data.comment += f' + missing({atom.atomtype.label}-{neighbors})'
-                    else:
-                        thermo_data.comment = f'Thermo group additivity estimation: ' \
-                                              f'missing({atom.atomtype.label}-{neighbors})'
+                    neighbors = ''.join(sorted([atom2.atomtype.label for atom2 in atom.edges.keys()
+                                                if atom2.atomtype.label != 'H']))
+                    neighbors += 'H' * len(['H' for atom2 in atom.edges.keys() if atom2.atomtype.label == 'H'])
+                    if atom.atomtype.label == 'Cb':
```

Comment:
Could this handling of Cb be confusing to a user since they might expect it to behave like the other labels?

Reply:
I thought I was following an RMG-db convention: https://github.com/ReactionMechanismGenerator/RMG-database/blob/master/input/thermo/groups/group.py#L49457

Reply:
Okay makes sense!